src/main/java/com/easypost/model/Brand.java

```diff
@@ -12,6 +12,7 @@
     private String name;
     private String userID;
     private String theme;
+    private String id;
```

Comment:
Please write up a ticket internally (Phab) to document the `id` field in our docs, we'll do that separate.